l/libxv/stone.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name       : libxv
+version    : '1.0.12'
+release    : 1
+homepage   : https://www.x.org/
+upstreams  :
+    - https://www.x.org/releases/individual/lib/libXv-1.0.12.tar.gz : ce706619a970a580a0e35e9b5c98bdd2af243ac6494c65f44608a89a86100126
+summary    : X11 Video extension library
+description: |
+    libXv provides an X Window System client interface to the XVideo extension to the X protocol.
+license    : MIT
+emul32     : yes
```

Comment:
This looks left over from whatever package you copied and pasted as your base. I doubt we need the 32bit libraries for this (at least not yet)

Reply:
Fixed